Libraries/src/Amazon.Lambda.Annotations.SourceGenerator/Writers/CloudFormationWriter.cs

```diff
@@ -324,5 +330,75 @@ private void ProcessLambdaFunctionRole(ILambdaFunctionSerializable lambdaFunctio
                 _templateWriter.SetToken($"{rolePath}.{GET_ATTRIBUTE}", new List<string>{role, "Arn"}, TokenType.List);
             }
         }
+
+        /// <summary>
+        /// Suffix that is appended to the CloudFormation template with the name
+        /// and version of the Lambda Annotations library
+        /// </summary>
+        public static string CurrentDescriptionSuffix
+        {
+            get
+            {
+                var version = Assembly.GetAssembly(MethodBase.GetCurrentMethod().DeclaringType).GetName().Version.ToString();
+                return $"{BASE_DESCRIPTION} (v{version}).";
+            }
+        }
+
+        /// <summary>
+        /// This appends a string to the CloudFormation template description field with the version
+        /// of Lambda Annotations that was used during compilation.
+        /// 
+        /// This string allows AWS to report on these templates to measure the usage of this framework.
+        /// This aids investigations and outreach if we find a critical bug, 
+        /// helps understanding our version adoption, and prioritizing improvements to this library 
```

Comment:
nit: the last sentence is grammatically incorrect. It should have a verb such as "aids" and "helps", which you've used in previous sentences. The last sentence shouldn't start with an "-ing".

Reply:
Agreed, fixed via amending the commit.